### PR TITLE
Rename vars

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

File: roles/jws/defaults/main.yml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 # installation method: ['local_zipfiles','rhn_zipfiles','zipfiles','rpm']
-tomcat_install_method: local_zipfiles
+jws_install_method: local_zipfiles
 
-tomcat_version: 9.0.60
-tomcat_zipfile_home: "{{ tomcat_install_dir }}/apache-tomcat-{{ tomcat_version }}"
-tomcat_zipfile: "apache-tomcat-{{ tomcat_version }}.zip"
-tomcat_native_zipfile: ''
-tomcat_zipfile_url: "https://archive.apache.org/dist/tomcat/tomcat-{{ tomcat_version.split('.')[0] }}/v{{ tomcat_version }}/bin/{{ tomcat_zipfile }}"
+jws_version: 9.0.60
```
Comment (csutherl) [inline roles/jws/defaults/main.yml:15]: This isn't a valid JWS version. I think we talked about there being an issue with updating default values automatically, but if we're going to call it jws_version it should have a valid JWS version :)

File: roles/jws/defaults/main.yml [hunk 2]
```diff
@@ -27,14 +27,14 @@ jws_rhn_base_url: https://access.redhat.com/jbossnetwork/restricted/softwareDown
...
+jws_group: tomcat
+jws_gid: 53
 
 # use default paths for tomcat or jws zipfiles depending on install_method; or can be overriden
-tomcat_home: "{{ jws_home if tomcat_install_method == 'rhn_zipfiles' else tomcat_zipfile_home }}"
-tomcat_setup: True
+tomcat_home: "{{ jws_home if jws_install_method == 'rhn_zipfiles' else tomcat_zipfile_home }}"
+jws_setup: True
 
 tomcat_catalina_base: "{{ lookup('env','CATALINA_BASE') }}"
```
Comment (csutherl) [inline roles/jws/defaults/main.yml:39]: Since we've already changed everything else, why not change this block too?

**Review by csutherl — CHANGES_REQUESTED** [review]
I don't see any changes to `roles/jws/vars/main.yml`? Shouldn't we be updating the variables in there too?

**Comment by rpelisse** [comment]
Yep, the default value for jws_version will disappear upstream( jws_version will be undefined upstream). 
Noted for the 39 to 49

File: roles/jws/vars/main.yml [hunk 3]
```diff
@@ -2,67 +2,66 @@
 tomcat:
```
Comment (csutherl) [inline roles/jws/vars/main.yml:2]: Do you not need a change here to from tomcat to jws?

**Review by csutherl — APPROVED** [review]
That's a big PR, but I think it LGTM :smile: Just had one commented about the `roles/jws/vars/main.yml` but it doesn't keep me from giving a +1.

File: roles/jws/vars/main.yml (hunk 3 above)
Comment (rpelisse) [reply roles/jws/vars/main.yml:2]: Yes, we do! And that's the next step :) (this PR being already quite huge)

**Pull request merged**